### PR TITLE
(Blog) pdksync tool

### PR DESCRIPTION
**Pull request by sheenaajay**
Request not to merge until everyone is happy with the doc and all comments are incorporated. Thank you.

### CODE REVIEW

**Comment by pmcmaw** [comment]
LGTM, although it may be worth while including how to add a tag as this is super useful for us when creating lots of PRs.

**Comment by sheenaajay** [comment]
> LGTM, although it may be worth while including how to add a tag as this is super useful for us when creating lots of PRs.

sure will add that too. Thank you @pmcmaw

File: _docs/pdksync.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+`bundle exec rake 'pdksync:run_a_command[touch dog]'`
+
+`bundle exec rake 'git:create_commit[removeboltgem, (maint) Remove bolt gem from sync.yaml file]'`
+
+`bundle exec rake 'git:push_and_create_pr[(MODULES-xxxx) Remove bolt gem from the modules]'`
+
+### Using scripts with pdksync
+
+Sometimes you will want to make many changes in each module. below is a simple bash script to execute 3 commands in each module
```
Comment (sanfrancrisko) [inline _docs/pdksync.md:43]:
```suggestion
Sometimes you will want to make many changes in each module. Below is a simple bash script to execute 3 commands in each module
```

File: _docs/pdksync.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+`bundle exec rake 'pdksync:run_a_command[ruby yamlbthere.rb Gemfile]'`
+`bundle exec rake 'pdksync:run_a_command[rm -f yamlbthere.rb]'`
+
+Executing ruby scripts is even more powerful. 
+
+Imagine the script below running against every module. 
+
+```
+require 'yaml'
+require 'pry'
```
Comment (sanfrancrisko) [inline _docs/pdksync.md:55]: Just a suggestion to remove the `require 'pry'` from the demo script

**Review by sanfrancrisko — COMMENTED** [review]
Just minor issues, but otherwise LGTM 👍

File: _docs/pdksync.md (hunk 2 above)
Comment (sheenaajay) [reply _docs/pdksync.md]: @sanfrancrisko Thank you

**Review by sanfrancrisko — APPROVED** [review]

**Pull request merged**